tests/python/test_ipython.ipynb

```diff
@@ -0,0 +1,66 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "51b3b00e",
+   "metadata": {},
+   "source": [
+    "# Count primes below a given bound"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "83062572",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import taichi as ti\n",
+    "\n",
+    "def test_kernel_print():\n",
```

Comment:
IIUC you only defined a kernel but didn't actually run it right?

Reply:
Doesn't pytest automatically collect all functions begin with `test_xxx` and run it? :thinking: :

Reply:
Yes, it didn't actually run the function. Should be fixed now.